{"identifier":{"url":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/RawValue-swift.typealias","interfaceLanguage":"swift"},"sections":[],"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"metadata":{"roleHeading":"Type Alias","fragments":[{"kind":"keyword","text":"typealias"},{"text":" ","kind":"text"},{"kind":"identifier","text":"RawValue"}],"role":"symbol","modules":[{"name":"FoundationX"}],"navigatorTitle":[{"text":"RawValue","kind":"identifier"}],"title":"URLError.NetworkUnavailableReason.RawValue","platforms":[{"beta":false,"deprecated":false,"introducedAt":"13.0","name":"iOS","unavailable":false},{"name":"macOS","introducedAt":"10.15","deprecated":false,"unavailable":false,"beta":false},{"deprecated":false,"unavailable":false,"beta":false,"introducedAt":"13.0","name":"tvOS"},{"unavailable":false,"name":"watchOS","introducedAt":"6.0","beta":false,"deprecated":false}],"externalID":"s:10Foundation8URLErrorV24NetworkUnavailableReasonO8RawValuea","symbolKind":"typealias"},"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"RawRepresentable.RawValue"},{"type":"text","text":"."}],"hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum"]]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum\/rawvalue-swift.typealias"]}],"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"text":"RawValue","kind":"identifier"},{"text":" = ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:Si","text":"Int"}]}],"kind":"declarations"}],"references":{"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/NetworkUnavailableReason-swift.enum/RawValue-swift.typealias":{"role":"symbol","navigatorTitle":[{"text":"RawValue","kind":"identifier"}],"fragments":[{"text":"typealias","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"RawValue"}],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/RawValue-swift.typealias","title":"URLError.NetworkUnavailableReason.RawValue","type":"topic","kind":"symbol","abstract":[],"url":"\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum\/rawvalue-swift.typealias"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError":{"title":"URLError","role":"symbol","url":"\/documentation\/foundationx\/urlerror","type":"topic","kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","abstract":[{"text":"Describes errors in the URL error domain.","type":"text"}],"navigatorTitle":[{"text":"URLError","kind":"identifier"}],"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"URLError"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX":{"abstract":[],"kind":"symbol","title":"FoundationX","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","url":"\/documentation\/foundationx","role":"collection","type":"topic"},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/NetworkUnavailableReason-swift.enum":{"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"NetworkUnavailableReason","kind":"identifier"}],"url":"\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum","kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum","role":"symbol","title":"URLError.NetworkUnavailableReason","abstract":[{"text":"Reasons used by URLError to indicate that a URLSessionTask failed because of unsatisfiable network constraints.","type":"text"}],"navigatorTitle":[{"kind":"identifier","text":"NetworkUnavailableReason"}]}}}